packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -37,17 +39,31 @@ class PropertyEditorController extends DisposableController
       _editableWidgetData;
   final _editableWidgetData = ValueNotifier<EditableWidgetData?>(null);
 
+  ValueListenable<bool> get shouldReconnect => _shouldReconnect;
+  final _shouldReconnect = ValueNotifier<bool>(false);
+
+  bool get waitingForFirstEvent => _waitingForFirstEvent;
+  bool _waitingForFirstEvent = true;
+
   late final Debouncer _editableArgsDebouncer;
 
+  late final Timer _checkConnectionTimer;
+
   static const _editableArgsDebounceDuration = Duration(milliseconds: 600);
 
+  static const _checkConnectionInterval = Duration(seconds: 5);
```

Comment:
do we need to check this frequently?

Reply:
Hmm it looks like no! It seems like the timer starts immediately after the tab is re-awoken, so even setting this to 1 minute means the overlay is displayed immediately after returning from sleep.